dist/components/app.jsx

```diff
@@ -53,40 +128,143 @@ const Book = styled.section`
   padding: 20px;
 `;
 
+const Button = styled.section`
+  margin-left: 2%;
+  margin-right: 2%;
+  border: solid;
+  color: white;
+  padding: 15px 50px 15px 50px;
+  background-color: #fc534e;
+  border-radius: 7px;
+  font-size: 16px;
+`;
+
 const ChargedYet = styled.section`
   text-align: center;
+  font-size: 13px;
+  margin-top: -20px;
+  font-weight: 500;
 `;
 
 const Bar = styled.section`
   border: solid;
-  border-width: thin;
-  margin-left: 2%;
-  margin-right: 2%;
+  border-width: .25px;
+  border-color: #D0D0D0
+  margin-left: 8%;
+  margin-right: 8%;
   margin-top: 10px;
   margin-bottom: 10px;
 `;
 
-const Footer1 = styled.section``;
+const Bar2 = styled.section`
+  border: solid;
+  border-width: .5px;
+  border-color: #D0D0D0
+  margin-left: 8%;
+  margin-right: 8%;
+  margin-top: 10px;
+`;
+
+const Bar3 = styled.section`
+  border: solid;
+  border-width: .5px;
+  border-color: #D0D0D0
+  margin-top: 10px;
+  margin-left: 3%;
+  margin-right: 3%;
+`;
+
+const Footer1 = styled.section`
+  margin-top: 20px;
+  margin-left: 8%;
+  font-weight: bold;
+  font-size: 14px;
+`;
 
-const Footer2 = styled.section``;
+const Footer2 = styled.section`
+  margin-top: 5px;
+  font-size: 14px;
+  margin-left: 8%;
+  margin-bottom: 20px;
+`;
 
 const LightBulb = styled.section``;
 
 const TotalPrice = styled.section`
   text-align: left;
+  margin-left: 10px;
+  margin-top: -10px;
+  font-size: 14px;
+  font-weight: lighter;
+  display: inline-block;
+  margin-right: 210px;
 `;
 
 const ServiceFee = styled.section`
   text-align: left;
+  margin-left: 8px;
+  margin-top: 12px;
+  font-size: 14px;
+  font-weight: lighter;
+  display: inline-block;
+  margin-right: 220px;
 `;
 
-const OccupancyFeeAndTaxes = styled.section`
+const OccupancyFeeAndTaxes = styled.section`  
   text-align: left;
+  margin-left: 8px;
+  margin-top: 12px;
+  font-size: 14px;
+  font-weight: lighter;
+  display: inline-block;
+  margin-right: 132px;
+`;
+
+const Fee1 = styled.section`
+  text-align: right;
+  margin-right: 10px;
+  display: inline-block;
+  font-size: 14px;
+  font-weight: lighter;
+`;
+
+const Fee2 = styled.section`
+  text-align: right;
+  margin-right: 10px;
+  display: inline-block;
+  font-size: 14px;
+  font-weight: lighter;
+  font-size: 14px;
+  font-weight: lighter;
+`;
+
+const Fee3 = styled.section`
+  text-align: right;
+  margin-right: 10px;
+  display: inline-block;
+  font-size: 14px;
+  font-weight: lighter;
 `;
 
 const Total = styled.section`
   text-align: left;
   padding-bottom: 10px;
+  margin-left: 10px;
+  margin-top: 7px;
+  font-size: 14px;
+  padding-bottom: 20px;
+  display: inline-block;
+  margin-right: 250px;
+  font-weight: bolder;
+`;
+
+const Fee4 = styled.section`
+  text-align: right;
+  margin-right: 10px;
+  display: inline-block;
+  font-size: 14px;
+  font-weight: bolder;
+  display: inline-block;
 `;
```

Comment:
That's one big app! Are you planning on breaking this up once its functional?

Reply:
i would like to, but most of this app is just styling and basic rendering of listing details.  imo making a separate module/component just to render simple data feels over the top.  

i may make a separate styling module in spite of my irrational persistence that keeping all the styling in module is best practice